core/data/src/main/java/com/google/samples/apps/nowinandroid/core/data/util/ConnectivityManagerNetworkMonitor.kt

```diff
@@ -35,18 +35,28 @@ import kotlinx.coroutines.flow.conflate
 class ConnectivityManagerNetworkMonitor @Inject constructor(
     @ApplicationContext private val context: Context
 ) : NetworkMonitor {
-    override val isOnline: Flow<Boolean> = callbackFlow<Boolean> {
+    override val isOnline: Flow<Boolean> = callbackFlow {
+        val connectivityManager = context.getSystemService<ConnectivityManager>()
+
+        // The callback's methods are invoked on changes to *any* network, not just the active
```

Comment:
Shouldn't we use Multiline comments (`\**`) instead of single-line comments for three lines of description? This way it will also make it easier to collapse and uncollapse the comment later.

Reply:
Fair point, done.

Reply:
Good spot. Fixed on `main`